tests/system/python/conftest.py

```diff
@@ -1367,19 +1367,21 @@ def pytest_configure():
     pytest.PKG_MGR = 'yum' if pytest.IS_REDHAT else 'apt'
 
 
-def restart_and_wait_for_fledge(fledge_url, wait_time, auth_token=None):
+def restart_and_wait_for_fledge(fledge_url, wait_time, auth_token=None, use_https=False):
     """ Restarts the Fledge service and waits until it becomes responsive
 
     Args:
         fledge_url (str): base fledge url
         wait_time (int): Seconds between retries
         auth_token (str): Authorization Token (Optional)
+        use_https (bool): Whether to use HTTPS for ping checks after restart (Optional)
```

Comment:
https_enabled (bool, optional): Whether to use HTTPS instead of HTTP. Defaults to False.